packages/ui/src/Radio.module.scss

```diff
@@ -0,0 +1,93 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.wrapper {
+  display: inline-flex;
+  // inline-flex changes vertical align compared to display: flex
+  // setting vertical-align allows us to easily use radio button
+  // inline e.g. next to the text
+  // Most common use case will be still the usage in a form where parent
+  // wrapper controls this.
+  vertical-align: bottom;
+  cursor: pointer;
+
+  @include c.typographyBodySmall;
+
+  // hide native checkbox
+  input {
+    height: 1px;
+    width: 1px;
+    opacity: 0;
```

Comment:
nit: It's probably not breaking anything but should we also get rid of default `margin`?

Reply:
@danad02 we can, but considering other properties it really makes no difference (visual nor semantic). TBH I've never seen it anywhere.

Reply:
Let's leave it as is then.